{"name": "test_add[错误]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 3000 == 4000", "trace": "self = <test_calNew.Testcal object at 0x000001DB2D76F6D8>\nget_cal = <HogwartsLagou05.HomeWorkYing.PytestSecond.calculator.Calculator object at 0x000001DB2D76F400>, get_add_datas = [1000, 2000, 4000]\n\n>   ???\nE   assert 3000 == 4000\n\nC:\\Users\\zhy\\LGHomeworkUpload\\PytestSecond\\test_calNew.py:11: AssertionError"}, "attachments": [{"name": "stdout", "source": "9384e11c-b353-4f37-83be-8eb62e957f0d-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_add_datas", "value": "[1000, 2000, 4000]"}], "start": 1627298403136, "stop": 1627298403137, "uuid": "49b03b50-8e8a-4c27-87af-bda7a01376d3", "historyId": "f86a9fc270f4aa33c10aee3ddc047855", "testCaseId": "f84e8a019fd56aab92aa2c8c158e88eb", "fullName": "test_calNew.Testcal#test_add", "labels": [{"name": "suite", "value": "test_calNew"}, {"name": "subSuite", "value": "Testcal"}, {"name": "host", "value": "DESKTOP-RLF34HD"}, {"name": "thread", "value": "24512-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calNew"}]}